src/audio/SDL_sysaudio.h

```diff
@@ -48,7 +48,17 @@
 #define DEFAULT_AUDIO_RECORDING_CHANNELS 1
 #define DEFAULT_AUDIO_RECORDING_FREQUENCY 44100
 
-#define AUDIO_SPECS_EQUAL(x, y) (((x).format == (y).format) && ((x).channels == (y).channels) && ((x).freq == (y).freq))
+SDL_FORCE_INLINE SDL_bool SDL_AudioSpecsEqual(const SDL_AudioSpec *a, const SDL_AudioSpec *b)
```

Comment:
Don't need to force inline, just make it static and let the compiler decide. As a separate question, should we remove SDL_FORCE_INLINE for SDL3?

Reply:
I was just trying to replace a macro (which of course always force-inlines) with something that was more readable, as this got unwieldy. I don't think the function call overhead is gonna murder us here, though, so I'll just change this to be a normal function in a C file.

Reply:
Fixed in 5102dc74b34c85d381630b4b4915bf92180cf24b.